src/envs/cartpole.jl

```diff
@@ -1,9 +1,28 @@
-
+module CartPoleEnv
 # Ported from: https://github.com/openai/gym/blob/996e5115621bf57b34b9b79941e629a36a709ea1/gym/envs/classic_control/cartpole.py
 # which has header:
 # 		Classic cart-pole system implemented by Rich Sutton et al.
 #		Copied from https://webdocs.cs.ualberta.ca/~sutton/book/code/pole.c
 
+using Reinforce: AbstractEnvironment
+using LearnBase: DiscreteSet
+using RecipesBase
+@static if VERSION >= v"0.7"
+  using Random: seed!
+else
+  seed! = srand
+end
```

Comment:
I think we already dropped the v0.6 support. So julia `using Random: seed!`

Reply:
Pushed a new commit to reflect this.